install-steps.md

```diff
@@ -512,35 +512,35 @@ sudo cp ./oc /usr/local/bin/oc
 oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
 ~~~
 
-## Retrieving the OpenShift Installer (GA Release)
+### Create RHCOS images cache (Optional)
```

Comment:
Should this be a separate section instead of removing the 'Retrieving..." ? From what I see here we are totally replacing the instructions versus adding an additional option of cacheing.

Reply:
Hi Roger, it seems I removed that section while doing the rebase... let me fix that.

Reply:
Should be fixed now.